### PR TITLE
Fix a crash when handling CloudProviderAccountStatus

### PR DESCRIPTION
**Pull request by Anandkumar26**
Signed-off-by: Anand Kumar <kumaranand@vmware.com>

### CODE REVIEW

File: pkg/controllers/cloud/account_poller.go [hunk 1]
```diff
@@ -231,13 +231,16 @@ func (p *accountPoller) doAccountPoller() {
 		p.log.Info("failed to get account", "account", p.namespacedName, "account", account, "error", e)
 	}
 
-	discoveredStatus, e := cloudInterface.GetAccountStatus(p.namespacedName)
+	discoveredStatus := cloudv1alpha1.CloudProviderAccountStatus{}
+	newStatus, e := cloudInterface.GetAccountStatus(p.namespacedName)
```
Comment (reachjainrahul) [inline pkg/controllers/cloud/account_poller.go:235]: just status as variable is fine

Comment (Anandkumar26) [reply pkg/controllers/cloud/account_poller.go:235]: ok

Comment (Anandkumar26) [reply pkg/controllers/cloud/account_poller.go]: Done

**Review by reachjainrahul — APPROVED** [review]
/LGTM

**Pull request merged**